### DISCUSSION TITLE
International Character Set

### DISCUSSION THREAD

**Discussion by pjchester** [dbeaver/dbeaver#12645] [Q&A]
Hi

Environment: Community Version 21.0.5.202105160657 On Windows 10.
Database connection: Teradata using default Teradata jdbc driver 
Problem:  International characters (umlauts etc) are not displaying correctly in the Data tab for a table or view.

Config for driver has been amended to include CHARSET UTF8 but this has made no difference (see attachment)
Unable to find details in documentation, what do I need to set and where?
![teradata](https://user-images.githubusercontent.com/37105618/119973573-8ebd1000-bfab-11eb-87e3-92cf0ac552ae.jpg)
Thanks
Paul

**Comment by LonwoLonwo** [comment]
Hello @pjchester 

Try to change it here Preferences->General->Workspace
![2021-05-31 15_39_41-DBeaver Enterprise 21 1 0 - vw_Employees TR_vw_Employees](https://user-images.githubusercontent.com/45152336/120194624-7f95c680-c226-11eb-8e76-9f9d4df21278.png)

**Reply by pjchester** [reply]
Thank you Anastasiya.  Unfortunately this has not worked.  The change has been applied and to be doubly sure, DBeaver closed and restarted.
![utf8-prods](https://user-images.githubusercontent.com/37105618/120303701-fedad700-c2c6-11eb-9dbc-962539e64705.jpg)

**Reply by LonwoLonwo** [reply]
I found similar question https://github.com/dbeaver/dbeaver/discussions/12193 and you can use workaround with dbeaver.ini file from it.
